05/components/Menu2.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class Menu2 extends React.Component{
+    render(){
+        return (
+            <ul>
+            {this.props.items.map((item) => {
+                return (
+                    <li>
+                        <a href ={item.url}>{item.text}</a>
+                    </li>
+                    )
+                })
+            }
+            </ul>
```

Comment:
Można też napisać osobną metodę np. `renderItems()`, która zwraca to co zaznaczyłem, a w tym miejscu ją wywołać tj. `<ul>{this.renderItems()}</ul>` - czytelniej będzie ;)